include/bluetooth/bluetooth.h

```diff
@@ -861,7 +864,9 @@ int bt_le_per_adv_set_data(const struct bt_le_ext_adv *adv,
  *
  *  Enabling the periodic advertising can be done independently of extended
  *  advertising, but both periodic advertising and extended advertising
- *  shall be enabled before any periodic advertisement data is sent.
+ *  shall be enabled before any periodic advertising data is sent.
+ *  Enabling the periodic advertising while it is already
```

Comment:
Wrong commit for this change. Also @joerchan states that this change is not necessary, please remove it.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/25326#discussion_r446103470

The random address is set by the host when doing undirected advertising. The only way to have the controller generate the random address is by selecting the IRK through the peer-address parameter in LE-adv-params-set, which makes it a directed advertiser.

Reply:
Yeah, that was a rebase gone wrong I think :) Should be fixed.